piccolo/apps/playground/commands/run.py

```diff
@@ -329,6 +330,16 @@ def run(
                 "port": port,
             }
         )
+    elif engine.upper() == "COCKROACH":
+        db = CockroachEngine(
+            {
+                "host": host,
+                "database": database,
+                "user": "root",
+                "password": "",
+                "port": 26257,
```

Comment:
Port is kind of problematic, because we do allow the user to pass it in as an arg. I think we need to make port a nullable argument, which defaults to `None`. And if not passed in, then it's `5432` for Postgres and `26257` for Cockroach.

Reply:
Yes, I think we should give the user the option to specify the port. So like you've done, except we can simplify it:

```python
def run(
    ...
    port: Optional[int] = None,
    ...
):
    if engine.upper() == "POSTGRES":
        db: Engine = PostgresEngine(
            {
                ...
                "port": port or 5432,
            }
        )
    elif engine.upper() == "COCKROACH":
        db = CockroachEngine(
            {
                ...
                "port": port or 26257,
            }
        )
```

Reply:
Ok, I'll change that. Thanks.